config/templates/wso2/filter/filters.conf.njk

```diff
@@ -0,0 +1,46 @@
+# Tomcat access
+#
+[FILTER]
+    Name modify
+    Match app.{{typeTag}}.access
+    Add event.kind event
+    Add event.category web
+    Add event.dataset wso2.access{% if tmplTomcatTestAccessIndex %}
+    @metadata.index {{ tmplTomcatTestAccessIndex }}{% endif %}
+
+# Parser filter for generic multiline core parsing
+#
+[FILTER]
+    Name parser
+    Match app.{{typeTag}}.generic
+    Key_Name message
+    Parser parser-filter-regex-generic-{{typeTag}}
+    Reserve_Data On
+    Preserve_Key On
+
+# App logs (generic)
+#
+[FILTER]
+    Name modify
+    Match app.{{typeTag}}.generic
+    Rename tomcat_time @timestamp
+    Add event.kind event
+    Add event.category web
+    Add event.dataset application.log
+    Add @metadata.timestampFormat YYYY-MM-DD-HH:mm:ss.SSS{% if shimDisableMultilineKey %}
+    Add log.file.path /unknown.log
+    Add event.sequence 0{% endif %}{% if tmplTomcatTestApplicationIndex %}
```

Comment:
May not be necessary to use the shim as carbon doesn't look multiline.

Reply:
I'll confirm if it's multiline.

Reply:
I do see multiline carbon logs on translate
(/sw_ux/wso2/wso2am/repository/logs/wso2carbon.log)